components/blocks/TwoColImageLeft.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <section class="temp desktop-container lg:grid grid-cols-2 my-20">
+    <div class="hidden lg:block">
+      <BaseImage
+        class="mt-6"
+        :src="block.image.filename"
+        alt="block.image.alt"
+      />
+    </div>
+
+    <div class="mx-6">
+      <BaseHeading tag="h2" size="h3" class="my-5">{{
+        block.heading
+      }}</BaseHeading>
+
+      <BaseImage
+        class="my-5 lg:hidden"
```

Comment:
All of these `my-5`s could be easier to manage by adding `flex space-y-5` to the parent div.

Small thing, but saves you a few keystrokes